sky/utils/ux_utils.py

```diff
@@ -17,6 +17,6 @@ def print_exception_no_traceback():
                 raise ValueError('...')
     """
     original_tracelimit = getattr(sys, 'tracebacklimit', 1000)
-    sys.tracebacklimit = 0
+    # sys.tracebacklimit = 1
```

Comment:
Remnant?

Reply:
Good catch! Restored the original code. Thanks!